core/src/main/resources/config/liquibase/valtimo-master.xml

```diff
@@ -44,6 +44,7 @@
     <include file="changelog/20180919-context-processes.xml" relativeToChangelogFile="true"/>
     <include file="changelog/20190719-email-notification-settings-changelog.xml" relativeToChangelogFile="true"/>
     <include file="changelog/20190828-email-settings-changed-changelog.xml" relativeToChangelogFile="true"/>
+    <include file="changelog/20220111-create-shedlock-table.xml" relativeToChangelogFile="true"/>
```

Comment:
The date format is not entirely correct and its not placed on the bottom of this list. 
But it won't break anything so I've still approved it